src/api/rooms/getRoomMembers.ts

```
@@ -47,8 +47,14 @@ export const getRoomMembers = async (roomId: number): Promise<RoomMembersRespons
   try {
     const response = await apiClient.get<RoomMembersResponse>(`/rooms/${roomId}/users`);
     return response.data;
-  } catch (error) {
+  } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)